app/src/main/java/fr/free/nrw/commons/customselector/ui/adapter/ImageAdapter.kt

```diff
@@ -202,55 +202,38 @@ class ImageAdapter(
                 defaultDispatcher,
                 uploadingContributionList,
             )
-            scope.launch {
-                val sharedPreferences: SharedPreferences =
```

Comment:
Was this part not useful? Does the `Show already handled pictures` still work as intended?